building/tracks/deprecated-exercises.md

```diff
@@ -0,0 +1,29 @@
+# Deprecated exercises
+
+Deleting an exercise is a destructive action it would delete all users' solutions to that exercise.
+Therefore exercises can never be deleted.
+Instead, we allow exercises to be deprecated.
+
+Deprecating an exercise has the following behaviours:
+
+- Users who have downloaded or started the exercise can access their existing solutions as normal
+- Users show have **not** downloaded or started it, will not see or start the exercise.
+
+To help with this, we have config rules that accompany a deprecated exercise
+Deprecated exercises must have the following set in `config.json`:
+
+- **`uuid`** (unchanged from when it was active)
+- **`slug`**
```

Comment:
I'd probably also add the unchanged note here too, as renaming the slug doesn't make much sense (while still technically possible)

```suggestion
- **`slug`** (unchanged from when it was active)
```